test-helpers/src/startFixture/webpack.ts

```diff
@@ -96,25 +95,32 @@ export const startWebpackFixture = (
     });
     const compiler = webpack(config);
 
-    // @ts-expect-error - webpack version mismatch
-    const server = new WDS(compiler, {
-      hot,
-      stats: logLevel,
-      noInfo: true,
```

Comment:
These were removed in `webpack-dev-server` v4.

Reply:
Pushed [`86f7733` (#1287)](https://github.com/vanilla-extract-css/vanilla-extract/pull/1287/commits/86f773361151a15073e1dcc552580636b7d2462c) to restore the logging level. Feel free to drop the commit if you disagree.

Reply:
LGTM. I tried with something similar but kept getting type errors. Honestly not too concerned with the fixture log output, but thanks 👍.